### PR TITLE
Adds generic search thumbnail retrieval

### PR DESCRIPTION
**Pull request by aarongundel**
Adds search thumbnails - to be implemented at the project level.

### CODE REVIEW

**Review by chiatt — CHANGES_REQUESTED** [review]
This looks great. Just a couple unused imports to remove and I think adding that setting to the project template would be nice.

File: arches/settings.py [hunk 1]
```diff
@@ -66,6 +66,7 @@
 
 ELASTICSEARCH_HTTP_PORT = 9200  # this should be in increments of 200, eg: 9400, 9600, 9800
 SEARCH_BACKEND = "arches.app.search.search.SearchEngine"
+SEARCH_THUMBNAILS = False
```
[the remark this answers is not in the document]
Comment (aarongundel) [reply arches/settings.py:69]: done

File: arches/app/utils/search_thumbnail_fetcher_factory.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from logging import Logger
```
[the remark this answers is not in the document]
Comment (aarongundel) [reply arches/app/utils/search_thumbnail_fetcher_factory.py:1]: it is used on line 17

[the remark this answers is not in the document]
Comment (chiatt) [reply arches/app/utils/search_thumbnail_fetcher_factory.py:1]: On 17 you are using 'logging' which is defined on line 8 imported from line 2.  Logger itself is unused.

**Review by chiatt — APPROVED** [review]

**Pull request merged**